test/parallel/test-v8-flag-type-check.js

```diff
@@ -0,0 +1,6 @@
+var common = require('../common');
+var assert = require('assert');
+var v8 = require('v8');
+
+assert.throws(function() {v8.setFlagsFromString(1)}, TypeError, 'Bad argument');
+assert.throws(function() {v8.setFlagsFromString()}, TypeError, 'Bad argument');
```

Comment:
The third argument is not actually checked by assert.throws, it's only used in the error message when the assert fails.